android-sdk/src/main/java/org/sagebionetworks/bridge/android/util/retrofit/BodyOnSubscribe.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2016 Jake Wharton
```

Comment:
It's in the retrofit-adapters library but package-private. I'm stealing the code that lets you write a Retrofit service with Single, Completable, Observable, etc return types instead of Call'

We're using an auto-generated client, and currently don't know how much work it'd be to get it to generate these new responses and get handled by the adapter layer